CodeEdit.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -42,8 +42,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/CodeEditApp/CodeEditSymbols",
       "state" : {
-        "revision" : "8262a5de7504cba4fc7da033b24b7f679c634800",
-        "version" : "0.2.1"
+        "branch" : "optional-nsimage-symbol",
```

Comment:
Should we make a release on CodeEditSymbols rather than relying on a branch>?

Reply:
Oh good catch, I initially did that to test it to see if it would work. It did and I forgot to make a release and change this. I will do this now.

Reply:
Done ✅